modules/labs-react/text-input/lib/TextInput.Hint.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+import {
+  createComponent,
+  ErrorType,
+  ExtractProps,
+  styled,
+  StyledType,
+  Themeable,
+  useModelContext,
+} from '@workday/canvas-kit-react/common';
+import {space, type} from '@workday/canvas-kit-react/tokens';
+import {Box} from '@workday/canvas-kit-labs-react/common';
+import {TextInputModelContext} from './TextInput';
+import {TextInputModel} from './useTextInputModel';
+
+export interface TextInputHintProps extends ExtractProps<typeof Box>, Themeable {
```

Comment:
Same here, remove the element attributes:
```suggestion
export interface TextInputHintProps extends ExtractProps<typeof Box, never>, Themeable {
```